### PR TITLE
disable verbose logging for integration tests on Travis

### PR DESCRIPTION
**Pull request by marten-seemann**
@lucas-clemente: Do you know why Chrome is dumping so much debug information into the Travis log? I disabled both `outWriter` and `errWriter` in `gexec.Start`, but that doesn't seem to change anything.

### CODE REVIEW

File: integrationtests/chrome_test.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func chromeTestImpl(version protocol.VersionNumber, url string, blockUntilDone f
 	}
 	utils.Infof("Running chrome: %s '%s'", getChromePath(), strings.Join(args, "' '"))
 	command := exec.Command(path, args...)
-	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+	session, err := gexec.Start(command, nil, nil)
```
Comment (lucas-clemente) [inline integrationtests/chrome_test.go]: I don't like this, I'd like to have the Chrome log if it fails. Why did you change this?

**Comment by lucas-clemente** [comment]
It's probably because Chrome starts subprocesses without forking. I don't think there's anything we can do about it, and tbh I don't really care. I only look at logs if something fails, and then it might be useful.

File: integrationtests/chrome_test.go (hunk 1 above)
Comment (marten-seemann) [inline integrationtests/chrome_test.go]: A desperate attempt to make Chrome shut up.

**Comment by marten-seemann** [comment]
It's a lot of spam in our logs. It's annoying to find what is failing in all the Chrome debug output.

**Comment by lucas-clemente** [comment]
I don't think there's a nice solution for that.

**Comment by marten-seemann** [comment]
@lucas-clemente: There is. The output was not coming from Chrome, but from the Xvfb. I fixed that now.

**Comment by marten-seemann** [comment]
@lucas-clemente: This is ready for merging now. Please review.

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ before_install:
   - export GOARCH=$TRAVIS_GOARCH
   - go env # for debugging
   - "export DISPLAY=:99.0"
-  - "Xvfb $DISPLAY &"
+  - "Xvfb $DISPLAY > /dev/null 2>&1 &"
```
Comment (lucas-clemente) [inline .travis.yml]: You can use &> to redirect both stdout and stderr at the same time.

File: integrationtests/chrome/chrome_suite_test.go [hunk 3]
```diff
@@ -142,7 +142,7 @@ func chromeTestImpl(version protocol.VersionNumber, url string, blockUntilDone f
 	}
 	utils.Infof("Running chrome: %s '%s'", getChromePath(), strings.Join(args, "' '"))
 	command := exec.Command(path, args...)
-	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+	session, err := gexec.Start(command, nil, nil)
```
Comment (lucas-clemente) [inline integrationtests/chrome/chrome_suite_test.go]: I'm still not convinced dropping Chrome logs is the right thing to do, but if you insist…

Comment (marten-seemann) [reply integrationtests/chrome/chrome_suite_test.go]: The problem is that Ginkgo dumps the output of the gexec [errWriter](https://godoc.org/github.com/onsi/gomega/gexec#Start) if we run the tests with -v.

**Pull request merged**